--- TiMidity++-2.15.0/timidity/timidity.c.orig	2018-08-29 01:33:39.000000000 +0200
+++ TiMidity++-2.15.0/timidity/timidity.c	2021-12-22 21:44:01.522822393 +0100
@@ -5587,7 +5587,11 @@ MAIN_INTERFACE int timidity_post_load_co
     if (ctl->id_character == 'A' && (ctl->flags & CTLF_DAEMONIZE))
     {
 	int pid = fork();
+	int fd;
 	FILE *pidf;
+	for(fd=0;fd<256;fd++)
+		if (isatty(fd))
+			close(fd);
 	switch (pid)
 	{
 	    case 0:		// child is the daemon
@@ -5602,6 +5606,7 @@ MAIN_INTERFACE int timidity_post_load_co
 		}
 		exit(0);
 	}
+	setsid();
     }
 #endif
 
