--- TiMidity++-2.15.0/configure.in.orig	2021-12-22 21:46:56.871872445 +0100
+++ TiMidity++-2.15.0/configure.in	2021-12-22 21:47:00.128521469 +0100
@@ -138,7 +138,7 @@ AC_ARG_WITH(default-path,
       pkgdatadir='/usr/local/share/timidity'
    fi
   ])
-tmpdefpath="`eval \"echo ${pkgdatadir}\"`"
+tmpdefpath="`eval echo ${pkgdatadir}`"
 AC_DEFINE_UNQUOTED(DEFAULT_PATH,"$tmpdefpath",place to install patches)
 AC_DEFINE(PKGDATADIR,DEFAULT_PATH,a compatibility matter. ignore it.)
 
@@ -152,7 +152,7 @@ AC_ARG_WITH(module-dir,
       pkglibdir='/usr/local/lib/timidity'
    fi
   ])
-tmplibdir="`eval \"echo ${pkglibdir}\"`"
+tmplibdir="`eval echo ${pkglibdir}`"
 AC_DEFINE_UNQUOTED(PKGLIBDIR,"$tmplibdir",place to install modules)
 
 AC_ARG_WITH(xaw-resource-prefix,
