diff -urN TiMidity++-2.0.2.orig/timidity.cfg TiMidity++-2.0.2/timidity.cfg
--- TiMidity++-2.0.2.orig/timidity.cfg	Thu Jan  1 01:00:00 1970
+++ TiMidity++-2.0.2/timidity.cfg	Mon Jul 19 05:55:58 1999
@@ -0,0 +1 @@
+dir /usr/share/GUSpatches
diff -urN TiMidity++-2.9.0.org/timidity/timidity.h TiMidity++-2.9.0/timidity/timidity.h
--- TiMidity++-2.9.0.org/timidity/timidity.h	Thu Mar  2 18:32:39 2000
+++ TiMidity++-2.9.0/timidity/timidity.h	Thu Mar  2 18:32:54 2000
@@ -28,7 +28,7 @@
 
 /* You could specify a complete path, e.g. "/etc/timidity.cfg", and
    then specify the library directory in the configuration file. */
-/* #define CONFIG_FILE "/etc/timidity.cfg" */
+#define CONFIG_FILE "/etc/timidity.cfg"
 #ifndef CONFIG_FILE
 
 #ifdef DEFAULT_PATH
