diff -uNr TiMidity++-2.13.0.orig/interface/xaw_i.c TiMidity++-2.13.0/interface/xaw_i.c
--- TiMidity++-2.13.0.orig/interface/xaw_i.c	2003-09-02 13:11:36.000000000 +0200
+++ TiMidity++-2.13.0/interface/xaw_i.c	2005-05-18 11:48:42.000000000 +0200
@@ -261,7 +261,10 @@
 String bitmapdir = XAW_BITMAP_DIR;
 Boolean arrangetitle,savelist;
 static char **current_flist = NULL;
-static int voices = 0, last_voice = 0, voices_num_width;
+#ifndef ___PLAYMIDI_H_
+static int voices = 0;
+#endif /* ___PLAYMIDI_H_ */
+static int last_voice = 0, voices_num_width;
 static int maxentry_on_a_menu = 0,submenu_n = 0;
 #define OPTIONS_WINDOW 1
 #define FLIST_WINDOW 2
